flocker/acceptance/testtools.py

```diff
@@ -331,10 +331,18 @@ def log_result(result):
 
     @wraps(function)
     def wrapper(self, *args, **kwargs):
+
+        serializable_kwargs = kwargs.copy()
+        for kwarg in kwargs:
+            try:
+                json.dumps(kwargs[kwarg])
+            except TypeError:
```

Comment:
`json.dumps`'s docstring only mentions `TypeError`s. I guess that we'll see quickly when there is an unhandled exception.